server_status/views_test.py

```diff
@@ -61,6 +63,32 @@ def test_view(self):
 
         self.assertTrue(resp["status_all"] == views.UP)
 
+    @data(
+        (None, views.DOWN, SERVICE_UNAVAILABLE),
+        (
+            "-----BEGIN CERTIFICATE-----MITX-----END CERTIFICATE-----",
+            views.DOWN,
+            SERVICE_UNAVAILABLE
+        )
+    )
+    @unpack
+    @override_settings(HEALTH_CHECK=['CERTIFICATE'])
+    def test_certificate_status_fail(self, certificate_loc, certificate_status, expected_status):
```

Comment:
Add `@freeze_time("2020-09-18")` to make sure test fails assuming certificate has expiration date > 30 days before that.